common/src/main/java/com/wynntils/wc/objects/item/WynnGearStack.java

```diff
@@ -0,0 +1,418 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.objects.item;
+
+import com.wynntils.core.webapi.WebManager;
+import com.wynntils.core.webapi.profiles.item.DamageType;
+import com.wynntils.core.webapi.profiles.item.ItemProfile;
+import com.wynntils.core.webapi.profiles.item.MajorIdentification;
+import com.wynntils.core.webapi.profiles.item.RequirementType;
+import com.wynntils.features.ItemStatInfoFeature;
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.utils.MathUtils;
+import com.wynntils.utils.StringUtils;
+import com.wynntils.wc.objects.item.render.HighlightedItem;
+import com.wynntils.wc.objects.item.render.HotbarHighlightedItem;
+import com.wynntils.wc.utils.IdentificationOrderer;
+import com.wynntils.wc.utils.WynnUtils;
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.Style;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.inventory.Slot;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import org.lwjgl.glfw.GLFW;
+
+public class WynnGearStack extends WynnItemStack implements HighlightedItem, HotbarHighlightedItem {
```

Comment:
Oh, and can we please not drop the `Item` from `ItemStack`? If anything, it's the `Stack` that should be dropped -- I'd prefer `WynnGearItemStack` but if that is too long, then `WynnGearItem` is better than `WynnGearStack`. An itemstack is what we normally call an "item" in minecraft. A "stack" is a very generic concept.

Reply:
my logic for the name was mostly based on the differences between minecraft's `Item` and `ItemStack` classes, despite the ways that differs with the way minecraft items are normally talked about. we'll presumably have custom itemstacks for stackable items down the road, where the "stack" aspects actually become relevant and just calling them `WynnSomethingItem` wouldn't be as appropriate, in my opinion

Reply:
Yes, I agree, there is a difference between "item" and "itemstack" in Minecraft terminology. But, if we want to keep that distrinction, then just keep calling "itemstacks" "itemstacks", mmmkay..? :)